float64[] time
float64[] timeVelocityPlot
float32[] distance 
float32[] velocity

string type
int32 motorPower
